src/test/java/website/frontrow/util/CollisionHandlerTest.java

```diff
@@ -170,7 +174,7 @@ public void getNextPositionTestRetricted()
      * Some special box of terribleness.
      */
     @Test
-    public void getNextPositionTestSlightlyRetricted()
+    public void getNextPositionTestSlightlyRestricted()
```

Comment:
You preface the other tests methods with test<name>, shouldn't you also do that here?